client/data/users/use-delete-user-mutation.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
+import { useMutation, useQueryClient } from 'react-query';
+
+/**
+ * Internal dependencies
+ */
+import wp from 'calypso/lib/wp';
+
+function useDeleteUserMutation( siteId ) {
+	const queryClient = useQueryClient();
+	const mutation = useMutation(
+		( { userId, variables } ) =>
```

Comment:
This only has the `reassign` option AFAIK. Have you considered simplifying the code to make this more obvious?

Reply:
Let's iterate on that as we go and develop habits on queries and mutations 👍🏻